src/pages/HomePage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Countries from '../components/countries/countries';
+import MainHeader from '../components/NavBar/MainHeader';
+
+const HomePage = ({ countries }) => (
+  <>
+    <MainHeader title="World Data" />
+    <Countries countries={countries} />
+  </>
+);
+
+HomePage.propTypes = {
+  countries: PropTypes.arrayOf.isRequired,
+};
```

Comment:
- [ ] Note that this is throwing a prop validation error in the console. To solve this kindly use a proper prop validation of an array of objects:point_down: 

```js
HomePage.propTypes = {
  countries: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
};
```